server/services/user.js

```diff
@@ -10,10 +10,15 @@ const DEVICE_SCHEMA = Joi.object().keys({
   contactInformation: Joi.string()
 });
 
+function exists(userId) {
+  return User.exists(userId);
+}
+
 function createUser(userObject) {
   const userSchema = Joi.object().keys({
     name: Joi.string().required(),
-    email: Joi.string().email().required()
+    email: Joi.string().email().required(),
+    isSysAdmin: Joi.boolean()
```

Comment:
But this change is added to `createUser` not `updateUser`...where's the middleware stopping that? I see it on `updateUser` but not `create`